airflow/opt/providers/etna/etna/etls/linkers/single_cell_rna_seq.py

```diff
@@ -0,0 +1,113 @@
+"""
+Module container helper functions for processing metis files associated with both raw and processed single cell data,
+both in pools and non pools.  Notably, these functions are versioned to account for changes in the pipeline that may not
+necessaril ybe backported to all data sets.
+
+Most of these helpers are accessible on the main MetisEtlHelper class as methods.
+"""
+from typing import Mapping, Optional, List, Tuple
+
+import re
+
+from airflow.exceptions import AirflowException
+from airflow.models.xcom_arg import XComArg
+from airflow.utils.task_group import TaskGroup
+from etna.etls.metis import MatchedRecordFolder
+
+from etna import MetisEtlHelpers, link
+
+timepoint_regex = re.compile(r"^(?P<timepoint>[A-Z0-9]+-[A-Z0-9]+-[A-Z][A-Z]?\d+)[A-Z0-9]+.*")
+patient_regex = re.compile(r"^(?P<patient>[A-Z0-9]+-[A-Z0-9]+)-.*")
+day_regex = re.compile(r".*-[A-Z][A-Z]?(?P<day>\d+)$")
+
+# TODO:  Would be great if a naming service provided this structure for us.
+def create_and_link_parents(matches: List[MatchedRecordFolder]):
+    for match in matches:
+        if match.model_name == "sc_rna_seq_pool":
```

Comment:
Hm, will it be a problem that `coprojects_template` and I assume projects based off of it, use the model names `sc_seq` and `sc_seq_pool` without the `_rna` part?

Reply:
Good, point.  I can change this logic to look for `pool` or not `pool` to keep it generic.